scripts/pretest.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# interrupt on any error
+set -e
+
+# load .env file to get NODE_ENV
+if [ -f ".env" ]; then
+  source ".env"
+fi
+
+npm run hint
+
+if [ "$NODE_ENV" = "circleci" ] || [ "$NODE_ENV" = "development" ]; then
+  npm run rexec:api install_selenium_server.sh
```

Comment:
Because we only run tests locally or on circleCI, you don't need to load the `.env` and you don't need to test for the value of `NODE_ENV`. This simplifies this script to just run one command, which could be directly put in the `package.json`.

Also, `npm run exec:api install_selenium_server.sh` is a bit misleading.
Installing the selenium server should only apply to `website` and `app` (API doesn't have a user interface that needs to be tested). This command makes it sound like you are asking `api` to install selenium.
I'd do a npm script: `npm run test:e2e:install`.

One more simplification: you can merge `npm run test:e2e` and `npm run test:e2:exec` (so that it matches `npm run test:server`)

Sorry to bother you with all this but I think that keeping things simple with clear names makes it easier to understand how the system works. It will also make it easier for new people to understand what's going on (and help avoid bugs).